lib/token-assert.js

```diff
@@ -230,6 +230,7 @@ TokenAssert.prototype._setEmptyLineIndentation = function(lineNumber, newWhitesp
  *
  * @param {Object} options.token
  * @param {Object} options.nextToken
+ * @param {boolean} [options.moveWhitespaceToNextToken]
```

Comment:
i'm unhappy with the name of this option and how it fits into the overall api. I'm open to better suggestions.